Signal/src/view controllers/MessagesViewController.m

```diff
@@ -2030,10 +2037,9 @@ - (void)yapDatabaseModified:(NSNotification *)notification {
               self.thread = [TSGroupThread threadWithGroupModel:gThread.groupModel transaction:transaction];
           }
         }];
+        [self setNavigationTitle];
```

Comment:
This is the second key change - the group name may have changed so we need to ensure that view and model state are aligned.